node/src/main.rs

```diff
@@ -19,13 +26,23 @@ pub struct Cli {
 pub enum Command {
     /// Start the node
     Start {
-        #[arg(short, long, value_name = "FILE", default_value = config::CONFIG_FILENAME)]
+        #[arg(short, long, value_name = "FILE", default_value = NODE_CONFIG_FILE_PATH)]
         config: PathBuf,
     },
 
-    /// Generate genesis file
+    /// Generates a genesis file and associated account files based on a specified genesis input
+    ///
+    /// This function creates a new genesis file and associated account files at the specified output
+    /// paths. It checks for the existence of the output file, and if it already exists, an error is
+    /// thrown unless the `force` flag is set to overwrite it. The function also verifies the existence
+    /// of a configuration file required for initializing the genesis file.
```

Comment:
Here, I would say "This command" rather than "This function".

Also, we should probably update the last sentence as we done verify the existence of configuration file here.